### PR TITLE
weather fetch call

### PR DESCRIPTION
**Pull request by abbeyroyse13**
# Description

fetch call for weather

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# Testing Instructions

Please describe the tests required to verify your changes. Provide instructions so PR Tester can check functionality. Please also list any relevant details for your tests

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] My changes generate no new warnings or errors
- [ ] I have added test instructions that prove my fix is effective or that my feature works

### CODE REVIEW

File: scripts/data/WeatherProvider.js [hunk 1]
```diff
@@ -1 +1,17 @@
-import { settings } from "../Settings.js"
\ No newline at end of file
+//import { settings } from "../Settings.js"
```
Comment (BrysonMG) [inline scripts/data/WeatherProvider.js:1]: import is commented out

File: scripts/data/WeatherProvider.js [hunk 2]
```diff
@@ -1 +1,17 @@
...
+
+let weatherForecast = [];
+
+export const getWeather = () => {
+
+    return fetch(`http: //api.openweathermap.org/data/2.5/forecast?id=524901&appid=e78bff76cd169fd6088b48c66c3fd622`)
```
Comment (BrysonMG) [inline scripts/data/WeatherProvider.js:7]: http: //api.openweathermap.org/data/2.5/forecast?id=524901&appid=e78bff76cd169fd6088b48c66c3fd622

should be

http://api.openweathermap.org/data/2.5/forecast?id=524901&appid=${settings.weatherKey}

**Review by BrysonMG — APPROVED** [review]
👍

**Review by MegMcGregor — APPROVED** [review]

**Review by jrossgage — APPROVED** [review]
Cool!

**Pull request merged**